scripts/cards-generator/src/camera/globals.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
+FOCAL_LENGTH = 50
+APERTURE = 24
```

Comment:
aperture is used to mean aperture (I assume film gate aperture) and sensor size in the code. Maybe use "sensor size" throughout?

Reply:
good call, fixed